xds/server_options.go

```diff
@@ -58,9 +59,10 @@ type ServingModeChangeArgs struct {
 	Err error
 }
 
-// BootstrapContentsForTesting returns a grpc.ServerOption which allows users
-// to inject a bootstrap configuration used by only this server, instead of the
-// global configuration from the environment variables.
+// BootstrapContentsForTesting returns a grpc.ServerOption with the default
+// pool set with the provided bootstrap congiguration. It allows users to
+// inject a bootstrap configuration to be used by only this server, instead of
+// the global configuration from the environment variables.
```

Comment:
We cannot change the bootstrap configuration used by a pool after creation. So, this server option should create a pool with the specified bootstrap configuration and pass that to the server which will use it to create the xDS client.

Reply:
Done